pulley/src/lib.rs

```diff
@@ -0,0 +1,183 @@
+//! The pulley bytecode for fast interpreters.
```

Comment:
Mind adding `#![cfg_attr(docsrs, feature(doc_auto_cfg))]` to the crate here too?